cdap-ui/app/features/home/routes.js

```diff
@@ -51,7 +51,20 @@ angular.module(PKG.name+'.feature.home')
           if(!n.length) {
             mySessionStorage.get(PREFKEY)
               .then(function (latest) {
-                var d = latest || rNsList[0].name;
+
+                var def = $filter('filter')(rNsList, {name: 'default'}, true);
```

Comment:
The third parameter is only useful when the 2nd parameter is a string, right?
Otherwise, substring match doesn't make sense, right?